### ISSUE TITLE
Logical OR

### ISSUE THREAD

**Issue by KhalilKhunji** [loly1710/js-functions-lab.#1]
https://github.com/loly1710/js-functions-lab./blob/dff1a2dbaa7a523fcb9eec125d3d12780ee92b8b/script.js#L60

Unfortunately, with the way you wrote this condition using the Logical OR (||) operator, this function will always return true. If you try running this code with different arguments (such as 'd', 'y', 1; anything, it can even be empty) it will return true.

This is because if you recall in class, OR checks if any one of the values is true/truthy, and if so returns the first truthy value.

What I'm assuming you wanted to do is this:
`if (letter === 'a' || letter === 'e' || letter === 'i' || letter === 'o' || letter === 'u')`

Which is a bit more cumbersome, but otherwise you run into the problem of it looking at the non-empty strings and resolving to true.

It might be easier to visualize like this:
letter ===  'a' || 
'e'  ||  (non-empty string: truthy)
'i' || (non-empty string: truthy)
'o' ||  (non-empty string: truthy)
'u' (non-empty string: truthy)

So the condition is not actually checking if the letter is equal to 'e', 'i', 'o' or 'u' in this case; it thinks you're asking it to see if the actual strings 'e', 'i', 'o' or 'u' are truthy or falsy. 

For more, consult the MDN page for Logical OR in Javascript: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR

**Comment by loly1710** [comment]
Hello Khalil

Oh! I already forgot to correct this mistake before submitting it, but I
know the correct way just like you wrote it for me now ..

Will my grades be deducted? 😞

**Comment by KhalilKhunji** [comment]
No, we don't grade your work in terms of marks. So long as we saw you made a serious attempt, your homework is considered as completed.

No need to worry about getting it right or wrong, just do your best to learn and get things submitted on time.
